RNTester/js/examples/PlatformColor/PlatformColorExample.js

```diff
@@ -0,0 +1,258 @@
+import {IOSDynamicColor} from '../../../../Libraries/StyleSheet/NativeColorValueTypesIOS';
+import {AndroidColor} from '../../../../Libraries/StyleSheet/NativeColorValueTypesAndroid';
+
+type State = {};
+
+class PlatformColorsExample extends React.Component<{}, State> {
+  state: State;
+
+  createTable() {
+    let colors = [];
+    if (Platform.OS === 'ios') {
+      colors = [
+        // https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors
+        // Label Colors
+        'labelColor',
+        'secondaryLabelColor',
+        'tertiaryLabelColor',
+        'quaternaryLabelColor',
+        // Fill Colors
+        'systemFillColor',
+        'secondarySystemFillColor',
+        'tertiarySystemFillColor',
+        'quaternarySystemFillColor',
+        // Text Colors
+        'placeholderTextColor',
+        // Standard Content Background Colors
+        'systemBackgroundColor',
+        'secondarySystemBackgroundColor',
+        'tertiarySystemBackgroundColor',
+        // Grouped Content Background Colors
+        'systemGroupedBackgroundColor',
+        'secondarySystemGroupedBackgroundColor',
+        'tertiarySystemGroupedBackgroundColor',
+        // Separator Colors
+        'separatorColor',
+        'opaqueSeparatorColor',
+        // Link Color
+        'linkColor',
+        // Nonadaptable Colors
+        'darkTextColor',
+        'lightTextColor',
+        // https://developer.apple.com/documentation/uikit/uicolor/standard_colors
+        // Adaptable Colors
+        'systemBlueColor',
+        'systemBrownColor',
+        'systemGreenColor',
+        'systemIndigoColor',
+        'systemOrangeColor',
+        'systemPinkColor',
+        'systemPurpleColor',
+        'systemRedColor',
+        'systemTealColor',
+        'systemYellowColor',
+        // Adaptable Gray Colors
+        'systemGrayColor',
+        'systemGray2Color',
+        'systemGray3Color',
+        'systemGray4Color',
+        'systemGray5Color',
+        'systemGray6Color',
+      ];
+    } else if (Platform.OS === 'android') {
+      colors = [
+        '?attr/colorAccent',
+        '?attr/colorBackgroundFloating',
+        '?attr/colorButtonNormal',
+        '?attr/colorControlActivated',
+        '?attr/colorControlHighlight',
+        '?attr/colorControlNormal',
+        '?android:colorError',
+        '?android:attr/colorError',
+        '?attr/colorPrimary',
+        '?colorPrimaryDark',
+        '@android:color/holo_purple',
+        '@android:color/holo_green_light',
+        '@color/catalyst_redbox_background',
+        '@color/catalyst_logbox_background',
+      ];
+    }
+
+    let table = [];
+    for (let color of colors) {
+      table.push(
+        <View style={styles.row} key={color}>
+          <Text style={styles.labelCell}>{color}</Text>
+          <View
+            style={{
+              ...styles.colorCell,
+              backgroundColor: PlatformColor(`${color}`),
```

Comment:
In previous conversations we've talked about having a lint rule that disallows non-statically analyzable values here.

So instead of having an array of strings where you pass the strings in, the array would be:

```
[
  PlatformColor('?attr/colorAccent'),
  PlatformColor('?attr/colorBackgroundFloating'),
  ...
```

There is a future direction for StyleSheet merging with our StyleX library where methods like this have to be statically analyzable so that they can be pulled out at build time to generate a css file. For mobile, these could be compiled out at build time and replaced with some other identifier and shared, if that was valuable. 

If we don't enforce this constraint at the beginning it will be very difficult to enforce it later. I'd rather decide down the line that we don't need that constraint anymore and remove it then.

Reply:
Got it.   Done.